frontend/src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
```

Comment:
remember to change the target branch from `main` to some feature branch name of your choosing since this feature doesn't work completely yet :)